src/Init/Data/BitVec/Bitblast.lean

```diff
@@ -267,6 +267,38 @@ theorem add_eq_adc (w : Nat) (x y : BitVec w) : x + y = (adc x y false).snd := b
 
 /-! ### add -/
 
+theorem getMsbD_add {i : Nat} {i_lt : i < w} {x y : BitVec w} :
```

Comment:
Is there a reason that these theorems are not `simp` unlike the previous ones?

Reply:
i assumed this was not `simp` since `getLsbD_add` isn't either and they have the same structure

Reply:
Given the relative complex RHS I would feel this is indeed the safer choice.